docs/runner/assets_create_if_not_exists.rst

```diff
@@ -0,0 +1,46 @@
+.. _assets_create_if_not_exists_yamlref:
+
+Assets Create If not exists Story Runner YAML
+..............................................
+
+Creates an asset with associated location and attachments if an asset does not exist
```

Comment:
Location and attachments are optional. This makes it sound like they're always created. The text lower down already says location and attachments are optional so I don't think the reference is needed here.

Suggest: "Searches for an asset matching the given signature. Returns that asset if found, else create a new one with the given signature and attributes."

Reply:
fixed